bentoml/_internal/bento/bento.py

```diff
@@ -11,33 +11,46 @@
 import fs.osfs
 import pathspec
 import yaml
-from fs.base import FS
-from fs.copy import copy_dir, copy_file
+from fs.copy import copy_file
 from simple_di import Provide, inject
 
 from ...exceptions import BentoMLException, InvalidArgument
 from ..configuration import BENTOML_VERSION
 from ..configuration.containers import BentoMLContainer
+from ..models import ModelStore
 from ..store import Store, StoreItem
 from ..types import PathType, Tag
 from ..utils import cached_property
 from .env import BentoEnv
 
-if TYPE_CHECKING:  # pragma: no cover
-    from ..models.store import ModelInfo, ModelStore
+if t.TYPE_CHECKING:  # pragma: no cover
+    from fs.base import FS
+
+    from ..models import Model
     from ..service import Service
-    from ..service.inference_api import InferenceAPI
 
 logger = logging.getLogger(__name__)
 
 BENTO_YAML_FILENAME = "bento.yaml"
 BENTO_PROJECT_DIR_NAME = "src"
 
 
+def _Bento_set_fs(bento: "Bento", _: t.Any, new_fs: "FS") -> "FS":
```

Comment:
what's the reason for the second parameter here?

Reply:
That's the `attrs` converter spec; it takes the old object, the field being assigned, and the new value. The field being assigned in this case is always `.fs`, so I just ignored it.